kytos/core/link.py

```diff
@@ -5,23 +5,25 @@
 """
 import json
 import operator
-from collections import OrderedDict
+from collections import OrderedDict, defaultdict
 from functools import reduce
 from threading import Lock
 
 from kytos.core.common import EntityStatus, GenericEntity
 from kytos.core.exceptions import (KytosLinkCreationError,
                                    KytosNoTagAvailableError)
 from kytos.core.id import LinkID
-from kytos.core.interface import TAGType
+from kytos.core.interface import Interface, TAGType
 
 
 class Link(GenericEntity):
     """Define a link between two Endpoints."""
 
     status_funcs = OrderedDict()
     status_reason_funcs = OrderedDict()
-    _get_available_vlans_lock = Lock()
+    _get_available_vlans_lock = defaultdict(Lock)
+    endpoint_a_lock = Lock()
```

Comment:
If you could remove these unused locks when you were refactoring and adapting the triple lock

Reply:
Deleted them